.github/workflows/bump-version.yml

```diff
@@ -39,37 +39,18 @@ jobs:
       - name: Checkout out
         uses: actions/checkout@v3
         with:
-          ref: ${{ inputs.branch || github.ref }}
-      - name: Setup python
-        uses: actions/setup-python@v4
+          ref: ${{ inputs.branch }}
+          fetch-depth: 0
```

Comment:
is this about submodules or something?

Reply:
seems to be needed for the tags as far as i understand...